wagtail/admin/checks.py

```diff
@@ -235,3 +235,41 @@ def file_overwrite_check(app_configs, **kwargs):
         )
 
     return errors
+
+
+@register("myapp.checks.datetime_format_check")
```

Comment:
We probably don't want a name that says myapp. Something short and simple is best. See the others in this file for examples.